Core/OxidEE_Events.php

```diff
@@ -153,9 +153,8 @@ private static function _createOrderTransactionTable()
             `WDOXIDEE_TRANSACTIONID` varchar(32) NOT NULL,
             `WDOXIDEE_PARENTTRANSACTIONID` varchar(32),
             `WDOXIDEE_REQUESTID` varchar(32) NOT NULL,
-            `WDOXIDEE_ACTION` enum('purchase','authorization') NOT NULL DEFAULT 'purchase',
-            `WDOXIDEE_STATE` enum('success','error','pending'),
-            `WDOXIDEE_PAYMENTMETHOD` varchar(32) NOT NULL,
+            `WDOXIDEE_ACTION` enum('authorization','purchase') NOT NULL DEFAULT 'purchase',
+            `WDOXIDEE_STATE` enum('error','pending','success'),
```

Comment:
awaiting state still missing as in other pull requests

Reply:
We added the `awaiting` state in WDPD-24.

Reply:
Yes, we decided to move it into WDPD-24 for the other PR:
https://jira.parkside.at/browse/WDPD-24